docs/input/docs/ci-systems/github.md

```diff
@@ -0,0 +1,79 @@
+---
+Order: 60
+Title: GitHub
+Description: Running on gh-actions
+---
+
+## specifics
+
+* `Upload-Artifacts`-Task
+
+  Uploading artifacts is currently not supported in Cake.Recipe. Please use the actions/upload-artifacts GitHub Action
+
+## example config
+
+```yaml
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+      - "release/**"
+      - "hotfix/**"
+      - "feature/**"
+    tags-ignore:
+      - "*"
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2.2.0
+        with:
+          fetch-depth: 0
+
+      - name: Cache Tools
+        uses: actions/cache@v2
+        with:
+          path: tools
+          key: ${{ runner.os }}-tools-${{ hashFiles('recipe.cake') }}
+      
+      - name: Setup .NET Core 2.1
+        uses: actions/setup-dotnet@v1.5.0
+        with:
+          dotnet-version: 2.1.809
+      
+      - name: Setup .NET Core 3.1
+        uses: actions/setup-dotnet@v1.5.0
+        with:
+          dotnet-version: 3.1.107
+      
+      - name: .net SxS Unix
+        if: matrix.os != 'windows-latest'
+        run: rsync -a ${DOTNET_ROOT/3.1.107/2.1.809}/* $DOTNET_ROOT/
+      
+      - name: .net SxS Windows
+        if: matrix.os == 'windows-latest'
+        run: |
+          SET DOTNET_31_ROOT=%DOTNET_ROOT:3.1.107=2.1.809%
+          xcopy /s /y /d %DOTNET_31_ROOT% %DOTNET_ROOT%
+        shell: cmd
```

Comment:
These actions are for side-loading multiple .NET Core editions. As a general example, the side-loading should not be here.
ie, should be changed to:`
```suggestion      
      - name: Setup .NET Core 3.1
        uses: actions/setup-dotnet@v1.5.0
        with:
          dotnet-version: 3.1.107
```

Reply:
@AdmiringWorm I have not done much with gh-actions. Why is this setting up .NET Core needed at all? According to https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md some versions are installed already. Or am I interpreting that wrong?

Reply:
You are right, it does seem like they finally have most of the SDKs installed.
Previously you had to install it yourself (except for the absolutely latest), and do additional work if you wanted multiple SDKs.

If you would be able to test that building an Addin would work without the step, I got no problems removing the step entirely.